cmd/crosschainbank.go

```diff
@@ -0,0 +1,96 @@
+package cmd
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+)
+
+// crosschainBankCmd represents the command to get balances across chains
+func crosschainBankCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "crosschainbank",
```

Comment:
Lets make this a top level `crosschain` (`Aliases: []string{"cc", "kriskross", "cchain", "coolchain"}`) with subcommands for each of the modules below? Love the idea but think we need better command structure here.